.gitignore

```diff
@@ -0,0 +1,16 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
+/.idea/libraries
+/.idea/modules.xml
+/.idea/workspace.xml
+/.idea/navEditor.xml
```

Comment:
Remove .idea folder completely from the commit, recheck for any such unnecessary folders